src/services/ec2/schema.graphql

```diff
@@ -116,8 +116,10 @@ type awsEc2 @key(fields: "arn") {
   eip: [awsEip] @hasInverse(field: ec2Instance)
   networkInterfaces: [awsNetworkInterface] @hasInverse(field: ec2Instance)
```

Comment:
for EC2 we need to make a connection to iamRole using the iamInstanceProfile property.

Reply:
oh yes, I had missed it. Done!